src/core/utils/building/find-entrypoints.ts

```diff
@@ -125,6 +144,9 @@ export async function findEntrypoints(
     if (include?.length && !exclude?.length) {
       return include.includes(config.browser);
     }
+    if (disallowedEntrypointNames.includes(entry.name)) {
+      return false;
+    }
```

Comment:
Now that I think about it, putting this inside `findEntrypoints` is wrong. When the `.wxt` directory is updated, the `.wxt/types/paths.d.ts` will also be filtered. We should probably move the filter to the `rebuild` function to prevent that from happening...

But we also need to move the `include` and `exclude` checks to `rebuild` for the same reason.

Let's keep this as-is for the this PR, no changes required. Let's move the filtering to `rebuild` in a separate, future PR.

Reply:
> But we also need to move the `include` and `exclude` checks to `rebuild` for the same reason.

Yes, I also found this problem.